index.bs

```diff
@@ -1696,22 +1696,78 @@ Team Representative in an Interest Group</h5>
+	Prior to [=AC Review=],
+	<dfn>charter drafts</dfn> go through a public [=charter refinement=] phase
+	during which the [=charter=] receives [=wide review=]
+	and the [=Chartering Facilitator=] attempts to find [=consensus=] on the charter.
+
+<h3 id="charter-initiation">
+Initiating Charter Refinement</h3>
+
+	[=Charter refinement=] (see below) is initiated
+	by the [=Team=] sending an [=charter/Advance Review Notice=]
+	to the [=Advisory Committee=],
+	to the public,
+	and, in the case of rechartering, to the affected [=Group=].
+
+	This <dfn for=charter>Advance Review Notice</dfn> must indicate:
+	* A short summary of the proposal.
+	* The location of the [=charter draft=], which must be public.
+	* How to participate in the discussion of this [=charter draft=] and where to file issues.
+	* The expected duration of the [=charter refinement=] phase,
+		which must not be less than 28 days,
+		and should not be more than 6 months.
+	* Who the [=Chartering Facilitator=] is.
+
+	The [=Team=] is responsible for initiating [=charter refinement=]
+	at its discretion, in consideration of discussions with the community.
+	An [=Advisory Committee representative=] <em class=rfc2119>may</em> formally request
+	that the [=Team=] initiate [=charter refinement=].
+	The Team <em class=rfc2119>may</em> deny such a request
+	if it thinks the proposal is insufficiently mature or does not align with W3C's scope and mission.
+	This rejection is a [=Team Decision=] which can be <a href="#registering-objections">Formally Objected to</a>.
+
+<h3 id="charter-development" oldids="WGCharterDevelopment">
+Charter Refinement</h3>
+
+	During <dfn>charter refinement</dfn>,
+	the W3C community further develops the [=charter draft=]
+	with the goal of achieving [=consensus=] on the proposal.
+	This process is chaired by the <dfn>Chartering Facilitator</dfn>,
+	who acts as [=Chair=] for [=charter refinement=].
+
+	Note: The [=Chartering Facilitator=] is not necessarily (and often is not) the [=Chair=] of the [=chartered group|group=] being chartered.
+
+	During [=charter refinement=]:
+	* All issues filed against the [=charter draft=] must be [=formally addressed=],
+		and their resolutions tracked in a disposition of comments.
+	* [=Wide review=] of the [=charter draft=] is initiated and completed.
+	* Decisions are made as  [=group decisions=] of all individuals participating in this process.
+		(However note that only W3C [=Members=] and the [=Team=] can particpate in a formal [[#Votes|vote]].)
+
+	The [=charter refinement=] phase concludes when there is either:
+	* A [=group decision=] to initiate [=AC Review=] of the [=charter draft=].
+	* A [=chair decision=] to abandon the proposal.
+
+	Any [=Formal Objection=] filed during the  [=charter refinement=] phase--
+	other than an objection to the choice of [=Chartering Facilitator=]
+	or to a decision to abandon the proposal--
```

Comment:
```suggestion
	Any [=Formal Objection=] filed during the [=charter refinement=] phase —
	other than an objection to the choice of [=Chartering Facilitator=]
	or to a decision to abandon the proposal —
```

Reply:
"--" at the end of the line gets automatically turned into em-dashes by the preprocessor we use (bikeshed): https://speced.github.io/bikeshed/#typography

Reply:
> * Ending a line with `--` will turn it into an em dash (`—`) and pull the following line upwards so there’s no space between the surrounding words and the dash.

Ugh. In my world, em dashes should be surrounded by spaces; en dashes and hyphens typically should not.

The W3C Style Guide points to the Chicago Manual of Style, and ["CMOS says no spaces, _but importantly their typography actually does have spacing around the em-dash,_ so basically they’re cheating a bit. AP says spaces."](https://www.reddit.com/r/FanFiction/comments/rac3rn/comment/hniu2wq/)

Reply:
Reported https://github.com/speced/bikeshed/issues/2834 and https://github.com/w3c/manual-of-style/issues/26 upstream. I don't really care which way we go, but I don't want to do ad hoc fixes for this, so using the bikeshed shorthand seems fine to me, and we should argue elsewhere about what that shorthand should resolve to.